proposals/configuration-level-resources.md

```diff
@@ -67,13 +79,380 @@ spec:
 The Kubernetes scheduler will schedule the Pod to the Node since the resources is available.
 The kubelet on that Node will make an `allocate` call to the `akri.sh/akri-onvif` device plugin in the Akri Agent.
 The Agent will then reserve slots in the associated Instance device plugins (`akri.sh/akri-onvif-8120fe` and `akri.sh/akri-onvif-a19705`).
-When selecting the slots from Instances, the Agent will preference unique Instances; for example, in the above Pod, the Agent will make sure 1 slot of each `akri-onvif-8120fe` and `akri-onvif-a19705` is reserved for each Pod.
+How the Agent select slots to reserve is based on the value of `Configuration.uniqueDevices`. 
+
+ When `Configuration.uniqueDevices` is true, the Agent will select max 1 slot for each Instance. For example, in the above Pod, the Agent will make sure 1 slot of each `akri-onvif-8120fe` and `akri-onvif-a19705` is reserved for each Pod.
```

Comment:
"each pod" here is confusing to me given that in the above scenario, you can have at most one broker. If you had two, that would mean four of `akri.sh/akri-onvif` would need to be allocated even though `Configuration.uniqueDevices=true` results in the following resources:

```
Capacity:
  akri.sh/akri-onvif:         2
  akri.sh/akri-onvif-8120fe:  2
  akri.sh/akri-onvif-a19705:  2
```